### DISCUSSION TITLE
Can you unzip a Self-extracting ZIP (universal) / Self-extracting ZIP?

### DISCUSSION THREAD

**Discussion by StephenMottershead** [gildas-lormeau/SingleFile#1615] [Q&A] [answered]
Hello, 

I am just testing things and I like the idea of Self-extracting ZIP / (universal) but sometimes I might want access to the larger images to view them. Is it at all possible to unzip this or am I better off use the normal zip format? 

I have tried using 7 zip, but it doesn't seem to recognize it as a zip.

**Comment by gildas-lormeau** [comment]
Hi, it should work with 7zip. Could you attach an example of file that you cannot unzip?

**Reply by StephenMottershead** [reply]
Hello, thank you for getting back you me. Not sure how to attach because GitHub didn't like the file format. The name for example - name.u.zip, and it opens as HTML. I have attached what happen when I right-click and go to 7zip.  
![Screenshot 2024-11-10 164428](https://github.com/user-attachments/assets/e9f64919-bbfa-4a31-b32b-e82ce7d91e73)

**Reply by gildas-lormeau** [reply, marked as answer]
You have to rename the file from `.zip.html` to `.zip` to unzip it.

**Reply by StephenMottershead** [reply]
Thank you. I needed to change a setting in Windows explorer, so I could edit the extension name. Untick "hide extension for known files types" in folder options, hope this helps someone else.
